README.md

```diff
@@ -23,9 +23,19 @@
 
 ## Being a better company to work at
 
+- [ ] Setup the ML cluster environment
```

Comment:
ML is not having a sperate cluster, they are using the same as everyone.

Reply:
Máximo, did you read the issue description which was linked?

Reply:
Cc @rporres @smola

Reply:
Witch description, I see here only one line.

Reply:
Damn it!!! It was discarded for some reason. https://github.com/src-d/feature-idea/issues/52

Reply:
Ok, this makes perfect sense, but then is "Setup GPU production cluster" and "Research and install Kubeflow (or alternatives) to the production clusters"